src/components/project.js

```diff
@@ -2,12 +2,14 @@ import React from "react"
 import Card from "react-bootstrap/Card"
 import ReactMarkdown from "react-markdown"
 import Linkify from "react-linkify"
-import Img from "gatsby-image"
```

Comment:
nah bring it back!

Reply:
Should I just add `import Img from "gatsby-image"` or add it and delete `import Container from "react-bootstrap/Container"`

Reply:
keep Container, we are still using that
re-introduce this import statement

then use _this_ component instead of the Card.Img component 

Basically re-introduce this line:
```<Img fluid={image} alt="" />```